### ISSUE TITLE
[completed-docs] Interface members are not checked

### ISSUE THREAD

**Issue by reduckted** [palantir/tslint#3534]
### Bug Report

- __TSLint version__: 5.8.0
- __TypeScript version__: 2.6.2
- __Running TSLint via__: CLI

#### TypeScript code being linted

```ts
/**
 * Some interface.
 */
interface SomeInterface {
    prop: number;
    method(): number;
 }
```

with `tslint.json` configuration:

```json
{
  "rules": {
    "completed-docs": [
      true,
      "interfaces",
      "methods",
      "properties"
    ]
  }
}
```

#### Actual behavior

No errors.

#### Expected behavior

`prop` and `method` should require documentation.

**Comment by reduckted** [comment]
This seemed like it would be a fairly simple fix - the rule wasn't checking `PropertySignature` or `MethodSignature` nodes. After adding cases to check those nodes, it started producing errors where the properties and methods didn't have comments, but it _also_ produced errors where the properties and methods _did_ have comments. 😕 

From what I could tell, `symbol.getDocumentationComment()` wasn't finding the documentation comments. This seems like an upstream issue with TypeScript, but maybe this could be solved if #3376 is implemented.

**Comment by felixfbecker** [comment]
This makes the rule pretty much useless to anyone who embraces functional class-free programming 🙁

**Comment by felixfbecker** [comment]
Since #3376 is a bit controversial, is there an upstream issue filed on TypeScript?

**Comment by reduckted** [comment]
Fixed in #4032 and released in [5.12.0](https://github.com/palantir/tslint/releases/tag/5.12.0)
